src/main/java/org/embulk/input/FtpFileInputPlugin.java

```diff
@@ -68,6 +71,7 @@
         @Config("port")
         @ConfigDefault("null")
         public Optional<Integer> getPort();
+        public void setPort(Optional<Integer> port);
```

Comment:
This method adding is ok. But I still don't know why it is added. It seems that this method is only used for saving the value of default port within newFTPClient method. Please correct me if no.